sources/packages/forms/src/form-definitions/sfaa2024-25.json

```diff
@@ -1920,7 +1920,7 @@
                     "strictDateValidation": false,
                     "multiple": false,
                     "unique": false,
-                    "onlyAvailableItems": false
+                    "onlyAvailableItems": true
```

Comment:
I believe that we will need to undo the changes for lists that have just one item present in the values. They are there only for UI and not data purposes. They are failing as shown below.

![image](https://github.com/bcgov/SIMS/assets/61259237/f89e0589-08e7-4f23-a0b8-7c12d661d3ac)